gulpfile.js

```diff
@@ -7,10 +7,10 @@ var requireDir = require('require-dir');
 var path = require('path');
 var debug = require('gulp-debug');
 
-var iusPath = path.join(process.cwd(), 'node_modules/gulp-ius-web');
+var iusPath = path.join(process.cwd(), 'node_modules/gulp-benefits-web/node_modules/gulp-ius-web');
 var ius = requireDir(iusPath);
 
-var appSettingsPath = path.join(process.cwd(), 'node_modules/gulp-appsettings');
+var appSettingsPath = path.join(process.cwd(), 'node_modules/gulp-benefits-web/node_modules/gulp-appsettings');
```

Comment:
this is a bug that we saw with Drew's machine but I thought that it was resolved.  To my knowledge this is not the way npm is supposed to work.